scilpy/io/utils.py

```diff
@@ -589,12 +591,23 @@ def is_header_compatible_multiple_files(parser, list_files,
 
     for filepath in list_files:
         _, in_extension = split_name_with_nii(filepath)
-        if in_extension not in ['.trk', '.nii', '.nii.gz']:
+        if in_extension not in ['.tck', '.trk', '.nii', '.nii.gz']:
             parser.error('{} does not have a supported extension'.format(
                 filepath))
 
     for curr in list_files[1:]:
-        if not is_header_compatible(list_files[0], curr):
+        compatible = True
+        if in_extension == '.tck':
+            # TCKs do not have a header, so compare with the reference's
+            # header.
+            _, curr_extension = split_name_with_nii(curr)
+            # If comparing two TCKs, the same reference will be used by
+            # both so they will be compatible.
+            if curr_extension != '.tck':
```

Comment:
So since only trk has a header I would reverse that condition, if it is .trk check it?